v2.1/sql-faqs.md

```diff
@@ -39,6 +39,30 @@ For example, this is how you’d use `RETURNING` to return an auto-generated [`S
 > INSERT INTO users (name) VALUES ('mike') RETURNING id;
 ~~~
 
+## What is transaction contention?
+
+Transaction contention occurs when transactions issued from multiple
+clients at the same time operate on the same data.
+
+This can cause transactions to wait on each other and decrease
+performance, like when many people try to check out with the same
+cashier at a store.
+
+For more information about contention, see [Understanding and Avoiding
+Transaction
+Contention](performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
+
+## Why can I not get more operations per second by increasing the number of nodes?
```

Comment:
I'd rephrase this a bit: `Why would increasing the number of nodes not result in more operations per second?`